tests/test_tree.py

```diff
@@ -33,6 +35,8 @@ def test_unserialized_tree_same_as_serialized(self):
 
         # Create some items that will modify
         tree = self.manager.tree
+        # Make sure we are working with a clean tree.
+        tree.clear()
```

Comment:
Why did you need to add this, did the test fail? It's collecting the session at the top, so it's normal that it is not empty.

Reply:
yep my mistake, I thought these were bleeding over from other tests, I didn't realize they were coming from the collection.